README.md

```diff
@@ -10,22 +10,28 @@ The dependency list is short: numpy and Cython (>= 0.21.1).
 Usage
```

Comment:
Also it'd be great to document our assumptions around the set of valid n-grams: can't start/end with n-grams, each token must be a uni-gram, etc.